.changeset/pink-turtles-sleep.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/wonder-blocks-labeled-field": patch
```

Comment:
**nit:** This would be a `minor` version, but I don't think it matters if you are planning to add more LabeledField PRs as part of this initial release.

Reply:
Updated it anyways to be concise! Good to know we want to use a minor update for setting attributes!

Side note - I've been adding changeset files to the intermediate PRs (these partX PRs are being merged into a `feature/labeled-field` branch). We only need one changeset when we merge to main, however, our pull requests check for a changeset file. I think it should be fine though that there will be multiple changeset files since it should result in just one version number change and consolidate the messages. Let me know what you think!

Reply:
That's true that we only need one changeset for main, but I feel that adding a changeset per PR helps to understand better the scope of the change for that PR. Agreed about having one version number + consolidated msgs. I guess using `minor` here is me being nitpicky 😅, but I guess I feel it's good to keep versions semantic just for keeping consistency with our PRs.

Maybe useful if we are looking back in history to look for specific changes, although I'm not sure if this PR will get squashed an erased as soon as we land the intermediate branch (`feature/labeled-field`) into `main`.

Reply:
Using `minor` here is a good call! I'm realizing now that none of the [other PRs I merged already](https://github.com/Khan/wonder-blocks/compare/main...feature/labeled-field) specified a minor change so at least there will be this one!

My plan is to land `feature/labeled-field` into `main` using `git land --feature-branch` so the PR merge commits don't get squashed (based on the [Intermediate Integration Branch example in Confluence](https://khanacademy.atlassian.net/wiki/spaces/ENG/pages/2918416504/Understand+our+Branching+Strategy#Example-2---Intermediate-Integration-Branch))